### PR TITLE
chore: commit generated types to the repository

### PR DESCRIPTION
**Pull request by dgozman**
This ensures that we get to see the generated types upon any changes,
since they are a part of our public api.

We update the types with Chromium roll, and validate the types
during lint, similar to api docs.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,13 +18,12 @@
...
-    "debug-test": "node --inspect-brk test/test.js",
-    "clean": "rimraf lib && rimraf types",
+    "lint": "npm run eslint && npm run tsc && npm run doc && npm run check-deps && npm run generate-types && npm run test-types && npm run test-infra",
+    "clean": "rimraf lib",
     "prepare": "node install-from-github.js",
-    "build": "node utils/runWebpack.js --mode='development' && tsc -p . && npm run generate-types",
+    "build": "node utils/runWebpack.js --mode='development' && tsc -p .",
     "watch": "node utils/runWebpack.js --mode='development' --watch --silent | tsc -w -p .",
-    "test-types": "npm run generate-types && npx -p typescript@3.7.5 tsc -p utils/generate_types/test/tsconfig.json && npm run typecheck-tests",
+    "test-types": "npx -p typescript@3.7.5 tsc -p utils/generate_types/test/tsconfig.json && npm run typecheck-tests",
```
Comment (JoelEinbinder) [inline package.json:26]:
```suggestion
    "test-types": "npm run generate-types && npx -p typescript@3.7.5 tsc -p utils/generate_types/test/tsconfig.json && npm run typecheck-tests",
```

Can we put generate-types back here? Not having it breaks my muscle memory for working with the types.

Comment (JoelEinbinder) [reply package.json:26]: Oh I see, it exits 1 if generating it fails.. That's unfortunate.

**Review by JoelEinbinder — APPROVED** [review]
Overall I don't like committing generated files. But if it helps you review, then it seems worth it. I would like some ability to run generate && test on the types.

**Comment by dgozman** [comment]
Decided to wait this this one.

**Pull request closed without merging**